tests/setuponlytests/test.sh

```diff
@@ -44,39 +44,37 @@ setupOnlyMinecraftTest(){
   # false positive since it's used in delta calculations below
   # shellcheck disable=SC2034
   start=$(date +%s)
+  status=PASSED
```

Comment:
I'm not able to see where `status` is set to "FAILED"

Reply:
You are right it was not